url/failure.html

```diff
@@ -16,7 +16,11 @@
 
     const name = test.input + " should throw"
 
-    self.test(() => { // new URL itself is already tested by url-constructor.html
+    self.test(() => { // URL's constructor's first argument is tested by url-constructor.html
+      assert_throws(new TypeError(), () => new URL("about:blank", test.input));
```

Comment:
It really seems like we should be only running these tests if test.base === "about:blank". But I guess it's probably true that if a URL fails to parse with a valid base, it will also fail to parse with about:blank?

Reply:
I think that should be true, yes.